imports/state/epics/unverified-warning.js

```diff
@@ -0,0 +1,28 @@
+// import { Accounts } from 'meteor/accounts-base'
+import { Meteor } from 'meteor/meteor'
+import { Subject } from 'rxjs/Subject'
+import { RESEND_VERIFICATION, RESEND_ERROR, RESEND_SUCCESS } from '../../ui/unverified-warning/unverified-warning.actions'
+
+import 'rxjs/add/operator/switchMap'
+import 'rxjs/add/operator/filter'
+
+export const unverifiedWarning = (action$, store) =>
+  action$.ofType(RESEND_VERIFICATION)
+    .filter(() => store.getState().resendVerificationState)
+    .switchMap(({email}) => {
+      const meteorResult$ = new Subject()
+      Meteor.call('resendEmail', Meteor.userId(), email, (err, result) => {
+        if (err) {
+          meteorResult$.next({
+            type: RESEND_ERROR,
```

Comment:
`RESEND_ERROR` doesn't seem to be ever in use by the UI. To save some implementation and design time, you can create a `genericErrorOccurred` action from *general-actions.js* and that will be dealt with a top level error dialog UI. (Once you do that remove all references the specific error type from other places)